geant4.spec

```diff
@@ -1,13 +1,13 @@
-### RPM external geant4 11.1.2
+### RPM external geant4 11.1.ref07
 ## INCLUDE compilation_flags
 ## INCLUDE compilation_flags_lto
 %define use_vecgeom 1
-#%define tag 35f304cde19a3e2dfe3bb941d4fc76c2fe45fa26
-#%define branch cms/v%{realversion}
-#%define github_user cms-externals
 %define tag cf4c6712c60cdf2b4d8799e34e176917c2de358d
-%define branch master
-%define github_user Geant4
+%define branch cms/v%{realversion}
+%define github_user cms-externals
+#%define tag cf4c6712c60cdf2b4d8799e34e176917c2de358d
+#%define branch master
+#%define github_user Geant4
```

Comment:
just update the value (tag and branch) without keeping the old values in commented out code